lib/errors.js

```diff
@@ -0,0 +1,16 @@
+export class FilecoinRpcError extends Error {
+  /**
+   * @param {string} message
+   * @param {string} method
+   * @param {string} name
+   */
+  constructor(name, message, method) {
+    super(message)
+    this.name = name
+    this.method = method
+  }
```

Comment:
Sub-classing errors in JavaScript is a bit tricky. Please add a call to `Error.captureStackTrace` as explained here:

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error/captureStackTrace#installing_stack_trace_on_a_custom_error_object